hpcbench/benchmark/osu.py

```diff
@@ -288,20 +287,20 @@ def arguments(self):
         """Dictionary providing the list of arguments for every
         benchmark"""
         if 'arguments' in self.attributes:
-            print(
+            logging.warning(
```

Comment:
Please use a dedicated logger instead of using the root logger. Do like `ior.py` for instance.